packages/commons/src/fhir/extension.ts

```diff
@@ -128,6 +128,10 @@ export type StringExtensionType = {
     valueString?: string
     valueBoolean: boolean
   }
+  'http://opencrvs.org/specs/extension/certificateTemplateId': {
+    url: 'http://opencrvs.org/specs/extension/certificateTemplateId'
+    valueString?: string
```

Comment:
Ok, in that case let's make it a non-optional string. We can make it optional when we notice there's a need for that. Otherwise to the next person reading the code, it will signal that there exists a condition where it can be optional

```suggestion
    valueString: string
```